Add support for BINDNOW_FLAGS to build artswrapper with non-lazy bindings, to
improve security and fix installation on FEATURES=stricter.
See bug #113934.

Requires the ebuild to export BINDNOW_FLAGS="$(bindnow-flags)".

Index: arts-1.5.0/soundserver/Makefile.am
===================================================================
--- arts-1.5.0.orig/soundserver/Makefile.am
+++ arts-1.5.0/soundserver/Makefile.am
@@ -78,7 +78,7 @@ artswrapper_SOURCES = artswrapper.c
 artswrapper_CFLAGS = $(KDE_USE_FPIE)
 artswrapper_INCLUDES = -I$(top_builddir) -DEXECUTE=\"$(artsdpath)\"
 artswrapper_LDADD = $(LIBPOSIX4)
-artswrapper_LDFLAGS = $(KDE_USE_PIE)
+artswrapper_LDFLAGS = $(KDE_USE_PIE) $(BINDNOW_FLAGS)
 
 artsshell_LDADD = libsoundserver_idl.la $(LIBPTHREAD)
 artsshell_LDFLAGS = $(USE_THREADS)
